esptfa_arima/Test_Management/services/analysis_doc_service.py

```
@@ -64,9 +66,18 @@ def create_analysis_document(draft: TestDraft):
         )
+
+        # get the specific objs from the json from the test_content
+        topics = draft.test_content['topics']
+        scores = draft.test_content['scores']
+
+        
         
-        # Process test topics
-        process_test_topics(document, draft.test_content)
+        # Process test topics and get the mappings
+        test_topics = create_test_topics(document, topics)
```

Ruff (0.14.10)

Undefined name `create_test_topics`

(F821)